app/src/lib/git/checkout.ts

```diff
@@ -87,13 +88,19 @@ export async function checkoutBranch(
           const description = progress.details.text
           const value = progress.percent
 
-          progressCallback({ kind, title, description, value, targetBranch })
```

Comment:
Just a nit: If we changed the `const targetBranch = branch.name` above to `const target = branch.name`, then this could remain one easy to digest line.

Reply:
I was wondering about it too. I went with 'minimal changes'.
Will change.
Atleast, it was not [this bad](https://thedailywtf.com/articles/changing-the-type-of-newbie).